runtime/oti/VMHelpers.hpp

```diff
@@ -2158,10 +2159,15 @@ class VM_VMHelpers
 	}
 
 	static VMINLINE void
-	cleanupContinuationObject(J9VMThread *vmThread, J9Object *objectPtr)
+	cleanupContinuationObject(J9VMThread *vmThread, J9Object *objectPtr, bool verifyOnly = false)
 	{
 #if JAVA_SPEC_VERSION >= 19
-		vmThread->javaVM->internalVMFunctions->freeContinuation(vmThread, objectPtr);
+		if (verifyOnly) {
```

Comment:
I don't think it's 'only'. you also have to free it up in verify mode, otherwise there is a leak

Reply:
for verify mode usage, we assume there is no case we can free up, so we assert when we find a case to free up.